geode/src/test/java/org/apache/calcite/adapter/geode/rel/GeodeZipsTest.java

```diff
@@ -45,6 +47,25 @@ public static void setUp() throws Exception {
     Cache cache = POLICY.cache();
     Region<?, ?> region =  cache.<String, Object>createRegionFactory().create("zips");
     new JsonLoader(region).loadClasspathResource("/zips-mini.json");
+    createTestRegion();
+  }
+
+  private static void createTestRegion() throws Exception {
```

Comment:
Can you pls create a separate test class for this ? I think `GeodeZipsTest` should only be used for zips model.